index.html

```diff
@@ -0,0 +1,983 @@
+                    </div>
+                    <div class="col-md-3 right">
+                        <a class="subscribe">
+                            <span>SUBSCRIBE</span> 
+                            <svg version="1.1" id="Layer_2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 8 14" style="enable-background:new 0 0 8 14;" xml:space="preserve"><g id="arrow-right"><defs><rect id="SVGID_1_" width="8" height="14"></rect></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_1_" style="overflow:visible;"></use></clipPath><g id="Shape_1" class="st0"><g><polygon class="st1" points="8,7 1.1,0 0,1.1 5.9,7 0,12.9 1.1,14 8,7 8,7	"></polygon></g></g></g></svg>
+                        </a>                    
+                        <a class="signIn">SIGN IN</a>
+    
+                    </div>
+                </div>
+                <div class="row navbarBottom">
+                    <div class="topRightSpan">
+                        <span></span>
```

Comment:
You can safely delete this node.

Reply:
![top-right](https://user-images.githubusercontent.com/29190411/51467998-2c65fc00-1d6e-11e9-9db8-5f0a01fee26f.PNG)

Reply:
I think that i do not want to delete this node...